docker-app/qfieldcloud/authentication/tests/test_authentication.py

```diff
@@ -80,6 +82,70 @@ def test_login_logout(self):
         self.assertEqual(len(tokens), 1)
         self.assertLess(tokens[0].expires_at, now())
 
+    def test_login_case_insensitive(self):
+        response = self.login("user1", "abc123")
+        tokens = self.user1.auth_tokens.order_by("-created_at").all()
+
+        self.assertEqual(len(tokens), 1)
+        self.assertTokenMatch(tokens[0], response.json())
+        self.assertGreater(tokens[0].expires_at, now())
+
+        response = self.login("USER1", "abc123")
+        tokens = self.user1.auth_tokens.order_by("-created_at").all()
+
+        self.assertEqual(len(tokens), 2)
+        self.assertTokenMatch(tokens[0], response.json())
+        self.assertGreater(tokens[0].expires_at, now())
+
+    def test_prevent_duplicate_case_insensitive_registration(self):
+        with self.assertRaises(django.db.utils.IntegrityError):
+            Person.objects.create_user(username="USER1", password="abc123")
+
+        with self.assertRaises(django.db.utils.IntegrityError):
+            Person.objects.create_user(username="uSeR1", password="abc123")
+
+        users = Person.objects.filter(username__iexact="user1")
+        self.assertEqual(users.count(), 1)
+
+    def test_case_sensitive_user_login_with_session(self):
```

Comment:
```suggestion
    def test_login_with_session_case_insensitive(self):
```

I believe we test the case insesitivity here, not sensitivity? I have suggested a name that is similar to `test_login_case_insensitive` above.

Reply:
Great!